<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusteriGirisi</class>
 <widget class="QMainWindow" name="MusteriGirisi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnaForm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnEkle">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>650</y>
      <width>83</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>EKLE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSil">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>730</y>
      <width>83</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>SİL</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtSoyad">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Adı</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ürün Bilgileri</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SoyAdı</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbwMusteriUrun">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>271</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnKaydet">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>83</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Kaydet</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtAdi">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>104</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBorc">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblBirimFiyat">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Borç (YTL)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>428</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
